test/bdd/fixtures/vault-server/.env

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# KMS
+KMS_IMAGE=ghcr.io/trustbloc-cicd/kms
+KMS_IMAGE_TAG=0.1.6-snapshot-f6e4b1f
+KMS_PORT=9097
+
+# EDV
+EDV_IMAGE=ghcr.io/trustbloc/edv
```

Comment:
we already have edv https://github.com/trustbloc/edge-service/tree/main/test/bdd/fixtures/edv-rest

Reply:
Done!